src/NuGet.Server/Web.config

```diff
@@ -22,6 +22,11 @@
     -->
     <add key="packagesPath" value="" />
 
+    <!--
+    Change the name of the internal cache file. Default is machine name (System.Environment.MachineName).
```

Comment:
It's unclear whether putting a path here works (or even _should_ work).

I would recommend updated the XML comment to state that this must be a file name and not a relative or absolute path. Additionally, I would validate at runtime that this is just a file name. In the future we could extend this to support relative paths but that seems unnecessary now and can be expanded without being a breaking change.

Reply:
Agreed. I updated the XML comment and added validation of a valid file name.